### PR TITLE
TF Chat Room

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Added the [TensorFlow Chat Room on Stackoverflow](https://chat.stackoverflow.com/rooms/216694/tensorflow) URL in the resources section.

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
I'm unsure where we draw the line on what we include in the README and what not. Adding wicke@ for policy

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Why add this image?

**Comment by martinwicke** [comment]
The initial commit (f5d859777b3340319d44cc4f8897cad9039d0ddd) is fine, although we should add something like "(not actively monitored by the TensorFlow team)". 

@theadactyl where do you think this pointer would be best? I'm actually thinking the community site: https://www.tensorflow.org/community/forums, rather than resources, which seems to be more about consumable content than interactive forums.

**Comment by martinwicke** [comment]
But yeah, the other two commits look like a mistake to me.

**Comment by Rishit-dagli** [comment]
@martinwicke @mihaimaruseac The other two commits are indeed a mistake, I messed up my TensorFlow fork branches while working on another contribution to the repo, I will fix this.

**Comment by Rishit-dagli** [comment]
@martinwicke @mihaimaruseac I have fixed the accidental commits. I apologize yet again for this confusion.

File: README.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ Build Type                                                        | Status
 *   [Intro to TensorFlow for Deep Learning from Udacity](https://www.udacity.com/course/intro-to-tensorflow-for-deep-learning--ud187)
 *   [Introduction to TensorFlow Lite from Udacity](https://www.udacity.com/course/intro-to-tensorflow-lite--ud190)
 *   [Machine Learning with TensorFlow on GCP](https://www.coursera.org/specializations/machine-learning-tensorflow-gcp)
+*   [TensorFlow Chat Room on StackOverflow](https://chat.stackoverflow.com/rooms/216694/tensorflow)
```
Comment (mihaimaruseac) [inline README.md:146]:
```suggestion
*   [TensorFlow Chat Room on StackOverflow (not actively monitored by the TensorFlow team)](https://chat.stackoverflow.com/rooms/216694/tensorflow)
```

**Review by mihaimaruseac — COMMENTED** [review]
Adding the suggestion that this is not something we actively monitor (at least now)

**Review by mihaimaruseac — APPROVED** [review]

**Comment by theadactyl** [comment]
LGTM

**Pull request merged**